internal/services/network/ip_group_resource.go

```diff
@@ -183,8 +183,25 @@ func resourceIpGroupRead(d *pluginsdk.ResourceData, meta interface{}) error {
 		}
 	}
 
-	d.Set("firewall_ids", getIds(resp.Firewalls))
-	d.Set("firewall_policy_ids", getIds(resp.FirewallPolicies))
+	var firewallIDs []string
```

Comment:
this'll default to `nil` rather than `[]string` fwiw, so we want to make this:

```suggestion
	firewallIDs := make([]string, 0)
```